main/src/CoffeeMachineSimulatorMain.java

```diff
@@ -0,0 +1,25 @@
+import java.util.Scanner;
+
+public class CoffeeMachineSimulatorMain {
+
+
+     public static String getCurrentLine;
+
+    //printing auctions, call myMethod
+    public static void main(String[] args) {
+        System.out.println("Write action (buy, fill, take, remaining, exit):");
+        getCurrentLine = getString();
```

Comment:
I think you need some loop to call getString function

Reply:
Hayk but all working

Reply:
I mean we need to create a program which is going to make latte espresso or whatever and finish when user set finish command yes ?

Reply:
Hayk jan i have a action method which when it see input"exit" is finishing program

Reply:
and in other cases too but, fill, remaining, e.t.c